Monitor/1.py

```diff
@@ -1,15 +1,21 @@
 import psutil
 from bottle import run, route
 
+# Static variable - stores number of requests processed
+count = 0
+
 @route('/stats',method = 'GET')
 def process():
+    global count
```

Comment:
Not these requests @rewanth1997, this is the monitoring request, we should count the requests sent to each vm at load balancing site.